guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -51,6 +57,43 @@ endif::[]
 +
 The script displays its progress and writes logs to `{installer-log-file}`.
 
+ifdef::katello,satellite,orcharhino[]
+. Optional: If you want to use an external PostgreSQL database in your {Project} deployment, configure your {ProjectServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--katello-candlepin-manage-db false \
+--katello-candlepin-db-host _postgres.example.com_ \
+--katello-candlepin-db-name candlepin \
+--katello-candlepin-db-user candlepin \
+--katello-candlepin-db-password _Candlepin_Password_ \
+--foreman-proxy-content-pulpcore-manage-postgresql false \
+--foreman-proxy-content-pulpcore-postgresql-host _postgres.example.com_ \
+--foreman-proxy-content-pulpcore-postgresql-db-name pulpcore \
+--foreman-proxy-content-pulpcore-postgresql-user pulp \
+--foreman-proxy-content-pulpcore-postgresql-password _Pulpcore_Password_ \
+--foreman-db-manage false \
+--foreman-db-host _postgres.example.com_ \
+--foreman-db-database foreman \
+--foreman-db-username foreman \
+--foreman-db-password _Foreman_Password_
+----
++
+To enable the Secure Sockets Layer (SSL) protocol for these external databases, append the following options:
```

Comment:
Perhaps we shouldn't emphasize SSL but emphasize encrypted connections? The technical implementation is SSL (or really, TLS these days) but I like focusing on the higher level goal.